torch/quantization/quantize.py

```diff
@@ -278,6 +293,7 @@ def quantize_qat(model, run_fn, run_args, inplace=False):
     prepare_qat(model, inplace=True)
     run_fn(model, run_args)
     convert(model, inplace=True)
+    _remove_qconfig(model)
```

Comment:
This will still cause errors, as convert (after training) will need a qconfig, so please remove it.